clang/test/Driver/sycl-host-compiler.cpp

```diff
@@ -27,13 +27,11 @@
 // RUN: %clangxx -fsycl -fsycl-host-compiler=g++ -E %s -### 2>&1 \
 // RUN:  | FileCheck -check-prefix=HOST_PREPROCESS %s
 // HOST_PREPROCESS: g++{{.*}} "-E"{{.*}} "-o" "[[PPOUT:.+\.ii]]"
-// HOST_PREPROCESS: g++{{.*}} "-E"{{.*}} "-o" "-"
```

Comment:
Is this going to require a rebase on #3645?

Reply:
Yes, a rebase is required.  #3645 looks to have been merged, so I will be updating this to reflect.